internal/pkg/service/buffer/worker/service/retry.go

```diff
@@ -0,0 +1,30 @@
+package service
+
+import (
+	"math"
+	"time"
+
+	"github.com/cenkalti/backoff/v4"
+)
+
+func NewRetryBackoff() *backoff.ExponentialBackOff {
+	b := backoff.NewExponentialBackOff()
```

Comment:
Presunute do suboru `retry.go`, kedze to okrem slice upload bude vyuzivat aj file import.